tests/FSharp.Data.Tests/WorldBankProvider.fs

```diff
@@ -1,7 +1,12 @@
-﻿#if INTERACTIVE
-#r "../../bin/FSharp.Data.dll"
+﻿#if NETSTANDARD2_0
+module FSharp.Data.Tests.WorldBankProvider
+do ()
```

Comment:
why id this do() here needed?

Reply:
it seems to be the only reason we need the #if NETSTANDARD2_0 at all on all these test files

Reply:
Great catch, I think that can be removed, trying it now

Reply:
I just went through all these test files and removed the netstandard2_0 if conditions. All tests still run and pass for me locally.